src/NLog/LogFactory.cs

```diff
@@ -1166,6 +1168,20 @@ public LogFactory LoadConfiguration(string configFile)
             return LoadConfiguration(configFile, optional: false);
         }
 
+        /// <summary>
+        /// Reload and apply the current config
+        /// </summary>
+        /// <returns>LogFactory instance for fluent interface</returns>
+        public LogFactory ReloadConfiguration()
+        {
+            InternalLogger.Info("Reloading configuration...");
+            Configuration = Configuration.ReloadNewConfig();
```

Comment:
Could this feature be made in a separate PullRequest?

Reply:
Will do